SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+SerenityOS is a hobby operating system developed by volunteers. It is not used anywhere in production (yet!), and so bugs and vulnerabilities can be safely disclosed publicly. If you find an issue, we would prefer if you report it as a [GitHub issue](https://github.com/SerenityOS/serenity/issues/new).
+
+We do care about security and encourage people to test it, and so we run a modest [Bug Bounty Program](https://serenityos.org/bounty/) for certain exploitable bugs. So if you find something exploitable, make sure to claim your reward. :^)
```

Comment:
Should we add a note to please check the oss fuzz tracker before reporting fuzzer bugs?

I remember a year or so ago we saw some people were copy pasting oss-fuzz bugs into the MITRE CVE tracker for questionable reasons.

Reply:
Yes please!